ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/logProcessor/leRobot/LeRobotDatasetEpisode.java

```diff
@@ -0,0 +1,227 @@
+            recorder.setFormat("mov");
+
+            // For information about these settings visit https://trac.ffmpeg.org/wiki/Encode/H.264
+            recorder.setVideoOption("preset", "ultrafast");
+            recorder.setVideoOption("crf", "27");
+            recorder.setVideoBitrate(60000000); // 6000 kb/s
+
+            // This video codec is deprecated, so in order to use it without errors we have to set the pixel format and strictly allow FFMPEG to use it
+            recorder.setVideoCodec(avcodec.AV_CODEC_ID_MJPEG);
+            recorder.setPixelFormat(avutil.AV_PIX_FMT_YUV420P);
+            recorder.setVideoOption("strict", "-2");
+            // Frame rate of video recordings
+            recorder.setFrameRate(15);
+
+//            recorder.setFormat("mp4");
+//            recorder.setVideoCodec(avcodec.AV_CODEC_ID_AV1);
+//            recorder.setPixelFormat(avutil.AV_PIX_FMT_YUV420P);
+////            float frameRate = sl_get_current_fps(zedSVOScrubber.getCameraID());
+////            LogTools.info("Recording at {} FPS", frameRate);
+//            recorder.setFrameRate(LeRobotDataset.ZED_FPS);
+
+            ExceptionTools.handle(() -> recorder.start(), DefaultExceptionHandler.MESSAGE_AND_STACKTRACE);
+
+            ffmpegRecorders.put(side, recorder);
+         }
+
+         long lastVideoTimestamp = -1;
+         int numberOfFrames = outPoint - inPoint;
+         long startVideoTimestamp = -1;
+         for (int i = 0; i < numberOfFrames; i++)
+         {
+            session.playbackTick(); // TODO: Is this skipping the first tick?
+
+            long timestamp = yoTimestamp.getLongValue();
+            zedSVOScrubber.scrub(timestamp);
+            long currentVideoTimestamp = zedSVOScrubber.getCurrentTimestamp();
+            if (startVideoTimestamp < 0)
+               startVideoTimestamp = currentVideoTimestamp;
+
+            if (currentVideoTimestamp > lastVideoTimestamp) // Write only when a new frame is available
+            {
+               lastVideoTimestamp = currentVideoTimestamp;
+
+               for (RobotSide side : RobotSide.values)
+               {
+                  Pointer zedColorImageSLMatPointer =
+                        side == RobotSide.LEFT ? zedSVOScrubber.getLeftColorImageSlMatPointer() : zedSVOScrubber.getRightColorImageSlMatPointer();
+                  Mat bgra8Mat = new Mat(imageHeight,
+                                         imageWidth,
+                                         opencv_core.CV_8UC4,
+                                         // BGRA8
+                                         sl_mat_get_ptr(zedColorImageSLMatPointer, SL_MEM_CPU),
+                                         sl_mat_get_step_bytes(zedColorImageSLMatPointer, SL_MEM_CPU));
+
+                  // Resize smaller for better transformer training
+                  Size size = new Size(853, 480);
+                  Mat resized = new Mat(size.height(), size.width(), opencv_core.CV_8UC4);
```

Comment:
You can pass in the size directly
```suggestion
                  Mat resized = new Mat(size, opencv_core.CV_8UC4);
```